corehq/toggles/models.py

```diff
@@ -0,0 +1 @@
+from toggle.models import Toggle, generate_toggle_id
```

Comment:
I didn't move the models in here yet to avoid breaking the cache for [Toggle.cached_get](https://github.com/dimagi/commcare-hq/blob/74bc31910f692126f03c46a350ab8ae5700f87dd/corehq/ex-submodules/toggle/models.py#L31), which caches an instance of Toggle.

Looking at this again and the comments on https://github.com/dimagi/commcare-hq/pull/30940/, I think I need to move the models into this file and just leave an import statement in the ex-submodules file. I'll add a commit to do that.